simplq/src/components/pages/Admin/ControlScroll.js

```diff
@@ -1,28 +1,26 @@
 // left: 37, up: 38, right: 39, down: 40,
 // spacebar: 32, pageup: 33, pagedown: 34, end: 35, home: 36
-var keys = {37: 1, 38: 1, 39: 1, 40: 1};
+let keys = {37: 1, 38: 1, 39: 1, 40: 1};
 
-function preventDefault(e) {
-  e.preventDefault();
-}
+const preventDefault = (e) => e.preventDefault();
 
-function preventDefaultForScrollKeys(e) {
+const preventDefaultForScrollKeys = (e) => {
   if (keys[e.keyCode]) {
     preventDefault(e);
     return false;
   }
 }
 
 // modern Chrome requires { passive: false } when adding event
-var supportsPassive = false;
+let supportsPassive = false;
 try {
   window.addEventListener("test", null, Object.defineProperty({}, 'passive', {
     get: function () { supportsPassive = true;  } 
   }));
 } catch(e) {}
 
-var wheelOpt = supportsPassive ? { passive: false } : false;
-var wheelEvent = 'onwheel' in document.createElement('div') ? 'wheel' : 'mousewheel';
+let wheelOpt = supportsPassive ? { passive: false } : false;
+let wheelEvent = 'onwheel' in document.createElement('div') ? 'wheel' : 'mousewheel';
 
```

Comment:
These variables should be declared using const I believe, not let, since they're not modified after creation.